examples/observability_at/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+resource "ibm_resource_group" "test_resource_group" {
+  name     = "${var.prefix}-rg"
+  quota_id = null
+}
```

Comment:
should we not use our resource group module so that existing resource groups can be used for the test?